db.go

```diff
@@ -390,10 +390,12 @@ type DB struct {
 	//
 	// The default is false, tracers should be registered by the callers.
 	OpentracingEnabled bool
-	context            context.Context
-	mu                 *sync.RWMutex // pointer, so copies in WithContext() behave properly.
-	models             map[reflect.Type]*Model
-	mappings           map[reflect.Type]fieldMap
+	// Span option functions are applied to every span started by squalor, when OpentracingEnabled is true.
+	SpanOptions []opentracing.StartSpanOption
```

Comment:
Does this need to be visible (capitalized)?

Reply:
nope - made private